include/aos/common/tools/fs.hpp

```diff
@@ -24,6 +24,44 @@ namespace aos {
  */
 constexpr auto cFilePathLen = AOS_CONFIG_FS_FILE_PATH_LEN;
 
+/**
+ * File system interface.
+ */
+class HostFSItf {
+public:
+    /**
+     * Gets mount point for the given directory.
+     *
+     * @param dir directory path.
+     * @return RetWithError<StaticString<cFilePathLen>>.
+     */
+    virtual RetWithError<StaticString<cFilePathLen>> GetMountPoint(const String& dir) const = 0;
+
+    /**
+     * Gets total size of the given directory.
+     *
+     * @param dir directory path.
+     * @return RetWithError<int64_t>.
+     */
+    virtual RetWithError<int64_t> GetTotalSize(const String& dir) const = 0;
```

Comment:
What is the difference between `GetTotalSize` and `GetDirSize`?

Reply:
`GetTotalSize` – Returns the total size of the file system (disk).
`GetDirSize``  – Returns the amount of space used by a directory.

Reply:
Done

Reply:
Done